user-guide/Cloud_Platform/AboutCloudPlatform/Connecting_your_DataMiner_System_to_the_cloud.md

```diff
@@ -4,75 +4,39 @@ uid: Connecting_your_DataMiner_System_to_the_cloud
 
 # Connecting your DataMiner System to the cloud
 
-> [!NOTE]
-> - This feature is in [soft launch](https://community.dataminer.services/documentation/soft-launch-options/) up to DataMiner 10.1.8. From DataMiner 10.1.9 onwards, it is fully available and no longer subject to change without notice. However, note that the Live Sharing feature remains in soft launch up to DataMiner 10.1.11.
-> - If you are using a DataMiner version between DataMiner 10.1.1 and DataMiner 10.1.9, and your system is **already connected** to the cloud, we recommend **installing the DataMiner Cloud Pack** (see step 2 below) on at least one DMA that was already hosting the cloud gateway. After you have done so, clear the selection of the checkboxes in System Center as detailed in the note under step 4 below.
-
 To connect your DMS to the cloud:
 
 1. Verify that the following **requirements** are met:
 
-    - All DataMiner Agents in the DataMiner System are running DataMiner version 10.1.1 or higher.
+    - All DataMiner Agents in the DataMiner System are running DataMiner version 10.1.12 or higher.
 
-    - Prior to DataMiner 10.1.9: The *CloudConnectedAgents* soft-launch option is enabled on all DMAs in the cluster. For more information, see <https://community.dataminer.services/documentation/soft-launch-options/>.
+    > [!Note]
+    > We recommend to upgrade to version 10.2 to optimally benefit from all the new features and capabilities. You can find the installer [here](xref:https://community.dataminer.services/downloads/).
 
-    - Each DMA that will be connected to the cloud can reach the following endpoints:
+     - Each DMA that will be connected to the cloud can reach the following endpoints:
```

Comment:
Remove space
```suggestion
    - Each DMA that will be connected to the cloud can reach the following endpoints:
```

Reply:
Changes have been implemented in a different commit